### PR TITLE
Suparna_Assignment_21_10min

### PR DESCRIPTION
**Pull request by SuparnaNikam**
maxnoand febonacci

### CODE REVIEW

File: src/suparnaNJune20/Assignment_21/MaxNoAndFibonacci.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	void maxNo(int no1, int no2) {
+		System.out.println("Before swap No1 = " + no1 + " No2 = " + no2);
+		no1 = no1 + no2;
+		no2 = no1 - no2;
+		no1 = no1 - no2;
+		System.out.println("After swap No1 = " + no1 + " No2 = " + no2);
+	}
+
+	void fibonacciSeries(int count) {
+		int no1 = 0, no2 = 1, no3, index;
```
Comment (VireshJJun20) [inline src/suparnaNJune20/Assignment_21/MaxNoAndFibonacci.java]: As per your current logic, Fibonacci series is starting from 0. 
Also, if value of n= 6, then ideally it should reflect 6 numbers. same when n=10 
you can try these changes:
int no1 = 1;

File: src/suparnaNJune20/Assignment_21/MaxNoAndFibonacci.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+		System.out.print(no1 + " " + no2);
+		for (index = 2; index <= count; ++index) {
```
Comment (VireshJJun20) [inline src/suparnaNJune20/Assignment_21/MaxNoAndFibonacci.java]: Check with this: 
for (index = 2; index < count; ++index) 

Rest logic looks good to me.

Comment (VireshJJun20) [inline src/suparnaNJune20/Assignment_21/MaxNoAndFibonacci.java]: Check with this: 
for (index = 2; index < count; ++index)

File: src/suparnaNJune20/Assignment_21/MaxNoAndFibonacci.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+Input n = 6
+Output = 1 1 2  3  5  8
+
+Input: n = 10 
+Output :  1 1 2  3  5  8 13  21 34 55
+
+ */
+package suparnaNJune20.Assignment_21;
+
+public class MaxNoAndFibonacci {
```
Comment (VireshJJun20) [inline src/suparnaNJune20/Assignment_21/MaxNoAndFibonacci.java:15]: Requirement is to create 2 separate programs.

Comment (SuparnaNikam) [reply src/suparnaNJune20/Assignment_21/MaxNoAndFibonacci.java:15]: created 2 separate program.

**Pull request merged**